### PR TITLE
feat(payments): PAYPAL-702 Add alternative to mapper

### PR DESCRIPTION
**Pull request by helga1507**
## What?
Add alternative to mapper

## Why?
New implementation alternative payment method in PayPal

## Testing / Proof
Unit, manual

ping @bigcommerce/payments @bigcommerce/checkout

### CODE REVIEW

File: src/payment/payment-method-ids.js [hunk 1]
```diff
@@ -5,5 +5,6 @@ export const BRAINTREE_VISACHECKOUT = 'braintreevisacheckout';
 export const BRAINTREE_GOOGLEPAY = 'googlepaybraintree';
 
 export const PAYPAL_COMMERCE = 'paypalcommerce';
+export const PAYPAL_COMMERCE_ALTERNATIVE_METHODS = 'paypalcommercealternativemethods';
```
Comment (BC-ACherednichenko) [inline src/payment/payment-method-ids.js:8]: We decided to put on hold this endpoint, for now, https://github.com/bigcommerce/bigcommerce/pull/37690/files#r507837862, @koa75

Comment (helga1507) [reply src/payment/payment-method-ids.js:8]: @BC-ACherednichenko It's different cases. This changes need for data to bigpay during submitting payment (`api/public/v1/orders/payments`) and the same gateway - "paypalcommerce"

Comment (BC-ACherednichenko) [reply src/payment/payment-method-ids.js:8]: ah, yes, please disregard in that case

**Review by BC-ACherednichenko — APPROVED** [review]
LGTM

**Pull request merged**